src/platform/web/ui/css/themes/element/theme.css

```diff
@@ -932,3 +978,87 @@ button.link {
     max-width: 200px;
     width: 100%;
 }
+
+/* Right Panel */
+
+.RoomInfo {
+    background: rgba(245, 245, 245, 0.90);
+    padding: 10px;
```

Comment:
I'd go for 16px here. Our designer recommended me to always use values multiple of 4px in any case, so you end up with a 4px grid that looks more coherent.